### ISSUE TITLE
[BUG] Payment succeeds / Webhook does not send

### ISSUE THREAD

**Issue by sutt** [lnbits/lnbits#1753]
**Describe the bug**

Invoice payment succeeds but fails to send webhook. 

UI displays sats are in wallet and payment succeeded, but webhook status displays "not sent yet" even after waiting minutes. Occurs for each invoice payment.

Logs display:
```
future: <Task finished name='Task-376' coro=<wait_for_paid_invoices() done, defined at /app/lnbits-legend/./lnbits/core/tasks.py:32> exception=AttributeError("'Payment' object has no attribute '_asdict'")>
    AttributeError: 'Payment' object has no attribute '_asdict'
```

Is this just a problem here https://github.com/lnbits/lnbits/commit/c960f718f5baf107cd2d9a079055138e5080b6f7 ? I can't find this `_asdict()` method defined anywhere else.

If not, I'll have to dig deeper into what's going on, but LND appears to be sending/receiving payments fine, just webhooks don't send.

**Desktop (please complete the following information):**

Running on Voltage with LNBits v0.10.8

**Comment by dni** [comment]
seeing this issue, aswell, investigation

**Comment by sutt** [comment]
@dni any update? did you confirm?

**Comment by talvasconcelos** [comment]
Fixed in #1757 !

**Comment by sutt** [comment]
Confirming we've got webhooks forwarding 🎉.
